src/platform/julius.c

```diff
@@ -16,6 +16,10 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#if defined(__SWITCH__)
```

Comment:
Suggest `#ifdef` since it's a single condition.

Reply:
fixed in 
https://github.com/bvschaik/julius/pull/124/commits/a9f9b4a6b45f5a4e2d9ced0b11791d0fe80a976c